resources/public/api/conf/1.2/application.yaml

```diff
@@ -394,9 +421,9 @@ paths:
                   type: string
                   example: 1a2b3c4d5e6f7g8h
                 payment_amount:
-                  description: The amount of money the parent or guardian wants to pay the childcare provider from their TFC balance.
-                  type: string
-                  example: £1234.56
+                  description: The amount of money the parent or guardian wants to pay the childcare provider from their TFC balance.  Supplied as 1/100 of the currency, i.e. Sterling pence. e.g. 20000 = £200.00.
+                  type: integer
```

Comment:
this was originally set as string.
should we stick to the string as everything else?

Reply:
According to the JIRA ticket, we should be changing to integer to match NS&I https://jira.tools.tax.service.gov.uk/secure/RapidBoard.jspa?rapidView=6625&projectKey=TFCA&view=detail&selectedIssue=TFCA-207